CMakeLists.txt

```diff
@@ -15,16 +15,17 @@ option(BUILD_FORTRAN "Build the Fortran Mex libraries" OFF)
 list(APPEND CMAKE_MODULE_PATH
     "${CMAKE_SOURCE_DIR}/cmake"
     "${CMAKE_SOURCE_DIR}/cmake/external"
+    "${CMAKE_SOURCE_DIR}/cmake/shared"
 )
 
 # Look for packages early so we can exit if they're not found
 include(herbert_FindMPI)
```

Comment:
why is herbert_FindMPI still project specific? You could remove the "herbert_core" dependence with an additional argument

Reply:
I did think about this, however we're not currently building anything against MPI in Horace, so thought it wasn't necessary at this point. It shouldn't be too difficult to make more general though, if you think there's some value in that